src/sql/workbench/contrib/notebook/browser/calloutDialog/imageCalloutDialog.ts

```diff
@@ -187,6 +177,19 @@ export class ImageCalloutDialog extends Modal {
 				ariaLabel: constants.embedImageLabel
 			});
 		DOM.append(embedRow, this._imageEmbedLabel);
+
+		this._register(this._imageRemoteRadioButton.onClicked(e => {
```

Comment:
Is the "online" functionality enabled yet?

Reply:
yes and it should work as expected.

Reply:
Please add a gif showing this functionality

Reply:
added it.